### ISSUE TITLE
Does not work: 'src/scrape_clipboard.py': [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by ariel777** [mperezi/alfred-git-clone#1]
Logs (when clipboards contains "https://github.com/mperezi/alfred-git-clone" ):
```
[15:02:51.139] Clone git repo[Keyword] Processing complete
[15:02:51.155] Clone git repo[Keyword] Passing output '' to Arg and Vars
[15:02:51.155] Clone git repo[Arg and Vars] Processing complete
[15:02:51.156] Clone git repo[Arg and Vars] Passing output '' to Script Filter
[15:02:51.179] Clone git repo[Script Filter] Queuing argument '(null)'
[15:02:51.292] Clone git repo[Script Filter] Script with argv '(null)' finished
[15:02:51.311] ERROR: Clone git repo[Script Filter] Code 2: /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'src/scrape_clipboard.py': [Errno 2] No such file or directory
```
Please have a look.

**Comment by mikybars** [comment]
Oops! That was my mistake when setting up the CI. I only included the contents of the `src` folder inside the workflow and so the paths were not correct. Fixed in [release v1.0.1](https://github.com/mperezi/alfred-git-clone/releases/tag/v1.0.1).

Thanks @ariel777 for your feedback!

**Comment by ariel777** [comment]
Works now! Thank you, @mperezi for fixing it!
